src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js

```diff
@@ -35,7 +35,7 @@ export const AzureComputeModalBase = ({
   const [viewMode, setViewMode] = useState(undefined)
   const [currentRuntimeDetails, setCurrentRuntimeDetails] = useState(() => getCurrentRuntime(runtimes))
   const [computeConfig, setComputeConfig] = useState(defaultAzureComputeConfig)
-  const updateComputeConfig = (key, value) => setComputeConfig(_.set(key, value, computeConfig))
+  const updateComputeConfig = (key, value) => setComputeConfig({ ...computeConfig, [key]: value })
```

Comment:
For other reviewers: I suggested this change - it's a bit easier to read and understand what's going on.

There's a bit of cleverness that goes on with _.set() that's additionally used in ComputeModal, which we will want to change as well.